app/src/main/java/com/kickstarter/ui/viewholders/discoverydrawer/ParentFilterViewHolder.java

```diff
@@ -43,7 +50,7 @@ public void bindData(final @Nullable Object data) throws Exception {
   public void onBind() {
     final Context context = context();
 
-    filterTextView.setText(item.params().filterString(context));
+    filterTextView.setText(item.params().filterString(context, ksString, false, true));
```

Comment:
Instead of injecting `ksString` in the constructor, you may also want to consider using `environment().ksString()`.

Tracing that through:

* We cast the context to a `KSApplication` object here: https://github.com/kickstarter/android-oss/blob/3d8b90965f494ccd61cd7a6ef9e68eded8e87e8e/app/src/main/java/com/kickstarter/ui/viewholders/KSViewHolder.java#L105
* `KSApplication`'s component gives us access to things injected from `ApplicationModule`, including the environment: https://github.com/kickstarter/android-oss/blob/master/app/src/main/java/com/kickstarter/ApplicationModule.java#L95

We're already using `environment` in view models, but we can use it elsewhere too. Eventually it'd be great to have Dagger build our dependency graph, but generally use `environment` to access globals similar to iOS since it makes test injection pretty straightforward.